storage/src/index/mod.rs

```diff
@@ -1393,8 +1555,14 @@ mod tests {
     fn test_partitioned_index_cursor_insert_before_next() {
         let runner = deterministic::Runner::default();
         runner.start(|context| async move {
-            let mut index = new_partitioned(context);
-            run_index_cursor_insert_before_next(&mut index);
+            {
+                let mut index = new_partitioned_unordered(context.clone());
+                run_index_cursor_insert_before_next(&mut index);
+            }
+            {
+                let mut index = new_partitioned_ordered(context.clone());
```

Comment:
We don't need clone on the context for the second test here?

Reply:
changed